src/unittest/python/plugins/python/integrationtest_plugin_tests.py

```diff
@@ -25,15 +25,40 @@
 from mock import patch
 
 from pybuilder.core import Project
-from pybuilder.plugins.python.integrationtest_plugin import (TaskPoolProgress,
-                                                             add_additional_environment_keys,
-                                                             ConsumingQueue)
+from pybuilder.plugins.python.integrationtest_plugin import (
+    TaskPoolProgress,
+    add_additional_environment_keys,
+    ConsumingQueue,
+    init_test_source_directory
```

Comment:
We should rename `init_test_source_directory` to `initialize_integrationtest_plugin`, it does not even remotely initialize directories